runit/php/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+mkdir /run/php/
+exec /usr/sbin/php-fpm7.4 --nodaemonize -R
```

Comment:
Not sure, but i assume this switches to some different user as configured (the workers).

Reply:
yes, in the end its all running with www-data:

```
www-data      92  0.0  0.0 228960  7740 ?        S    09:27   0:00 php-fpm: pool kopano-kdav
www-data      95  0.0  0.0 228960  7848 ?        S    09:27   0:00 php-fpm: pool kopano-kdav
www-data      96  0.0  0.0 228960  7848 ?        S    09:27   0:00 php-fpm: pool kopano-kdav
www-data      97  0.0  0.0 228960  7844 ?        S    09:27   0:00 php-fpm: pool kopano-kdav
www-data      98  0.0  0.0 228960  7844 ?        S    09:27   0:00 php-fpm: pool kopano-kdav
www-data     100  0.0  0.0 228960  7844 ?        S    09:27   0:00 php-fpm: pool kopano-kdav
www-data     103  0.0  0.0 228960  7848 ?        S    09:27   0:00 php-fpm: pool kopano-kdav
www-data     104  0.0  0.0 228960  7848 ?        S    09:27   0:00 php-fpm: pool kopano-kdav
www-data     110  0.0  0.0 228960  7848 ?        S    09:27   0:00 php-fpm: pool kopano-kdav
www-data     115  0.0  0.0 228960  7848 ?        S    09:27   0:00 php-fpm: pool kopano-kdav
www-data     116  0.0  0.0 228960  7848 ?        S    09:27   0:00 php-fpm: pool kopano-webapp
www-data     117  0.0  0.0 228960  7840 ?        S    09:27   0:00 php-fpm: pool kopano-webapp
www-data     118  0.0  0.0 228960  7844 ?        S    09:27   0:00 php-fpm: pool kopano-webapp
www-data     119  0.0  0.0 228960  7844 ?        S    09:27   0:00 php-fpm: pool kopano-webapp
www-data     120  0.0  0.0 228960  7844 ?        S    09:27   0:00 php-fpm: pool kopano-webapp
www-data     121  0.0  0.0 228960  7844 ?        S    09:27   0:00 php-fpm: pool kopano-webapp
www-data     123  0.0  0.0 228960  7848 ?        S    09:27   0:00 php-fpm: pool kopano-webapp
www-data     124  0.0  0.0 228960  7848 ?        S    09:27   0:00 php-fpm: pool kopano-webapp
www-data     125  0.0  0.0 228960  7852 ?        S    09:27   0:00 php-fpm: pool kopano-webapp
www-data     127  0.0  0.0 228960  7852 ?        S    09:27   0:00 php-fpm: pool kopano-webapp
www-data     128  0.0  0.0 229076  7852 ?        S    09:27   0:00 php-fpm: pool kopano-z-push
www-data     129  0.0  0.0 229076  7852 ?        S    09:27   0:00 php-fpm: pool kopano-z-push
www-data     130  0.0  0.0 229076  7852 ?        S    09:27   0:00 php-fpm: pool kopano-z-push
www-data     132  0.0  0.0 229076  7728 ?        S    09:27   0:00 php-fpm: pool kopano-z-push
www-data     133  0.0  0.0 229076  7852 ?        S    09:27   0:00 php-fpm: pool kopano-z-push
www-data     134  0.0  0.0 229076  7856 ?        S    09:27   0:00 php-fpm: pool kopano-z-push
www-data     135  0.0  0.0 229076  7856 ?        S    09:27   0:00 php-fpm: pool kopano-z-push
www-data     136  0.0  0.0 229076  7852 ?        S    09:27   0:00 php-fpm: pool kopano-z-push
www-data     137  0.0  0.0 229076  7852 ?        S    09:27   0:00 php-fpm: pool kopano-z-push
www-data     139  0.0  0.0 229076  7852 ?        S    09:27   0:00 php-fpm: pool kopano-z-push
```